parking_permits/services/traficom.py

```diff
@@ -149,7 +150,15 @@ def fetch_vehicle_details(self, registration_number):
         mass = et.find(".//massa")
         module_weight = mass.find("modulinKokonaismassa")
         technical_weight = mass.find("teknSuurSallKokmassa")
-        weight = module_weight if module_weight is not None else technical_weight
+        weight_et = module_weight if module_weight is not None else technical_weight
+        weight = int(weight_et.text) if weight_et.text else 0
```

Comment:
I would have some error handling for the` int()` call.

Reply:
Added safe casting to int.